app/api/user_management.py

```diff
@@ -0,0 +1,56 @@
+from flask import Flask, jsonify, request
+from flask_restplus import Resource, Api, fields
+try:
+    from app.models import UserManagement
+except Exception:
+    from WeConnect.app.models import UserManagement
+
+
+user_data = []
+# login_info = []
+
+
+UserObj = UserManagement()
+
+class RegisterUsers(Resource):
+    
+    
+    
+    def post(self):
+        
+        user_data = request.json
```

Comment:
You are not validating the data received from the client. Please try to incorporate this.